tt_eager/tt_dnn/op_library/transformer_tms/kernels/compute/ssm_eltwise_mul.cpp

```diff
@@ -128,6 +139,8 @@ void MAIN {
                 tile_regs_wait();
 
                 transpose_wh_init_short(cb_out_transposed);
+                unpack_reconfig_data_format(cb_in0_transposed, cb_out_transposed);
```

Comment:
`unpack_reconfig_data_format(cb_in0_transposed, cb_out_transposed);` This call reprograms both srcA & srcB, but transpose_wh does not need SrcB, you can use this:
```
/**
 * Helper function to reconfigure unpacker srca input data format, only if it differs from existing format.
 */
ALWI void unpack_reconfig_data_format_srca(const uint32_t srca_old_operand, const uint32_t srca_new_operand) {
    UNPACK(( llk_unpack_reconfig_data_format_srca(srca_old_operand, srca_new_operand) ));
}
```